packages/framework/src/Component/Cron/Config/CronConfig.php

```diff
@@ -33,16 +33,33 @@ public function __construct(CronTimeResolver $cronTimeResolver)
      * @param string $serviceId
      * @param string $timeHours
      * @param string $timeMinutes
+     *
+     * @deprecated Use `registerCronModuleInstance` instead
      */
     public function registerCronModule($service, $serviceId, $timeHours, $timeMinutes)
+    {
+        $this->registerCronModuleInstance($service, $serviceId, $timeHours, $timeMinutes, CronModuleConfig::DEFAULT_INSTANCE_NAME);
+    }
+
+    /**
+     * @param \Shopsys\Plugin\Cron\SimpleCronModuleInterface|\Shopsys\Plugin\Cron\IteratedCronModuleInterface $service
+     * @param string $serviceId
+     * @param string $timeHours
+     * @param string $timeMinutes
+     * @param string $instanceName
+     */
+    public function registerCronModuleInstance($service, string $serviceId, string $timeHours, string $timeMinutes, string $instanceName): void
```

Comment:
I am not sure about the naming...`registerCronModule` would fit perfectly, however, the name is taken...I have no idea how to name it but we can discuss it :slightly_smiling_face:

Reply:
A better name would be nice, but I didn't come up with anything clever.

Reply:
And what about not using the new method at all and just add the `string $instanceName = CronModuleConfig::DEFAULT_INSTANCE_NAME` parameter to the existing one? Or is it a bad practice :thinking:

Reply:
That would be a BC-Break. We agreed with @PetrHeinz, that I try to write this PR fully backward compatible.

Reply:
ok, lets keep the name, maybe we could discuss this topic on some meeting